### DISCUSSION TITLE
HTTPS not working with Symfony Server

### DISCUSSION THREAD

**Discussion by afilina** [symfony/symfony#47485] [Q&A]
### Symfony version(s) affected

6.1.0

### Description

I'm using Symfony CLI 5.4.13 and PHP 8.1.9
Attempts to use HTTPS gives a `Invalid request (Unsupported SSL request)` error in the Symfony CLI log.

### How to reproduce

I'm using Symfony CLI 5.4.13 and PHP 8.1.9

I ran `symfony server:ca:install` just like the documentation says. The console output `[OK]`. However, upon starting the server, it displays the HTTP address, and manually entering HTTPS gives the SSL error.

I tried to force renew a cert, but it had no effect. I also tried to reinstall Symfony CLI via cURL.

HTTPS used to work on this computer on older Symfony versions.

**Comment by MatTheCat** [comment]
This reminds me of https://github.com/symfony/symfony/discussions/47289 ; are you using Ubuntu?

**Reply by afilina** [reply]
Indeed. I'm not sure how to close this as duplicate, since the question remains unanswered.

**Reply by MatTheCat** [reply]
I'm not sure this is the same issue but Ubuntu may be a common factor 🤔

I will try Arch Linux when I can.

**Reply by MatTheCat** [reply]
No issue on Arch Linux. However, **two** URLs may be displayed: did you get a HTTPS one?

e.g.

```
[OK] Web server listening                                                                                              
      The Web server is using PHP CLI 8.1.10                                                                            
      https://127.0.0.1:8000                                                                                            
                                                                                                                        

[Web Server ] Sep  8 15:17:16 |DEBUG  | PHP    Reloading PHP versions 
[Web Server ] Sep  8 15:17:16 |DEBUG  | PHP    Using PHP version 8.1.10 (from default version in $PATH) 
[Web Server ] Sep  8 15:17:16 |INFO   | PHP    listening path="/usr/bin/php" php="8.1.10" port=41883
[PHP        ] [Thu Sep  8 15:17:16 2022] PHP 8.1.10 Development Server (http://127.0.0.1:41883) started
```

Here the correct URL is https://127.0.0.1:8000, not http://127.0.0.1:41883

**Reply by afilina** [reply]
Switched to Docker as a workaround, so this is no longer relevant to me. Thanks.
I remember trying the :8000 link too and it had the same outcome.
